### ISSUE TITLE
SkeletonUtils doesn't export it's members

### ISSUE THREAD

**Issue by sotrh** [three-types/three-ts-types#201]
-   `three` version:  ^0.137.0
-   `node` version: v16.13.0
-   `npm` (or `yarn`) version: yarn 1.22.15

### Problem description:

The `SkeletonUtils` namespace doesn't export any of it's functions. This means that Typescript will give the error "Property 'clone' does not exist on type 'typeof import("/home/ben/code/seek/compositor-engine/node_modules/@types/three/examples/jsm/utils/SkeletonUtils")'", even though the methods [are present and exported in THREE.js](https://github.com/mrdoob/three.js/blob/dev/examples/jsm/utils/SkeletonUtils.js).

These method's exist in the [type file](https://github.com/three-types/three-ts-types/blob/master/types/three/examples/jsm/utils/SkeletonUtils.d.ts), but they are not exported.

### Relevant code:

```ts
import * as SkeletonUtils from 'three/examples/jsm/utils/SkeletonUtils';

// ...

const clone = SkeletonUtils.clone(nodeToClone);
```

### Suggested solution:

As per the [doc on namespaces](https://www.tutorialsteacher.com/typescript/typescript-namespace), adding an export before each of the function, should do the trick.

**Comment by joshuaellis** [comment]
this will be resolved in `0.140.0`

**Comment by OswaldUnity** [comment]
Hello, I'm trying to use SkeletonUtils this way, but Rider nor VSCode detect the namespace and I can't get any intellisense
![image](https://user-images.githubusercontent.com/106346930/175136773-dcf3fb0d-ad31-4dd9-8132-e7a38c5713c4.png)

The SkeletonUtils work, but not only it doesn't show intellisense but give me an error as well.

What can I do to fix this?
